examples/13-streamlines/00_plot_3d_streamlines.py

```diff
@@ -0,0 +1,207 @@
+
+Compute and plot 3D streamlines
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example shows you how to compute and
+plot streamlines of fluid simulation results,
+for 3D models.
+
+"""
+
+###############################################################################
+# Compute and plot streamlines from single source
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+###############################################################################
+# Import modules, create the data sources and the model
+# -----------------------------------------------------
+# Import modules:
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+from ansys.dpf.core.helpers.streamlines import compute_streamlines
+from ansys.dpf.core.plotter import DpfPlotter
+
+###############################################################################
+# Create data sources for fluids simulation result:
+fluent_files = examples.download_fluent_mixing_elbow_steady_state()
+ds_fluent = dpf.DataSources()
+ds_fluent.set_result_file_path(fluent_files["cas"][0], "cas")
+ds_fluent.add_file_path(fluent_files["dat"][1], "dat")
+
+###############################################################################
+# Create model from fluid simulation result data sources:
+m_fluent = dpf.Model(ds_fluent)
+
+###############################################################################
+# Get meshed region and velocity data
+# -----------------------------------
+# Meshed region is used as geometric base to compute the streamlines.
+# Velocity data is used to compute the streamlines. The velocity data must be nodal.
+
+###############################################################################
+# Get the meshed region:
+meshed_region = m_fluent.metadata.meshed_region
+
+###############################################################################
+# Get the velocity result at nodes:
+velocity_op = m_fluent.results.velocity()
+fc = velocity_op.outputs.fields_container()
+field = dpf.operators.averaging.to_nodal_fc(fields_container=fc).outputs.fields_container()[0]
+
+###############################################################################
+# Compute and plot the streamlines adjusting the request
+# ------------------------------------------------------
+# The following steps show you how to create streamlines using DpfPlotter, with several sets
+# of parameters. It demonstrates the issues that can happen and the adjustments that you can make.
+
+###############################################################################
+# First, a DpfPlotter is created and the streamline is created with default values.
+pl0 = DpfPlotter()
+try:
+    streamline_obj = compute_streamlines(meshed_region=meshed_region, field=field)
+    pl0.add_mesh(meshed_region=meshed_region, opacity=0.3)
+    pl0.add_streamlines(streamlines=streamline_obj)
+    pl0.show_figure(show_axes=True)
+except:
+    # It throws an error and ends here, because source points are not set correctly.
```

Comment:
@anslpa do we need the try catch in the example?

Reply:
@cbellot000 I am assuming that a lot of people will face the issue that I faced, which was that the source was defined outside of the mesh geometry, and when trying to plot in this case, it is throwing. This try/except intends to demonstrate what's not expected to work. Is that an issue ?

Reply:
@anslpa can't we catch the error ourselves when trying to plot an empty streamline? in `add_streamlines`

Reply:
I think @PProfizi is right, and the issue you encountered should be added in the docstring but using try except in an example (failing secretly BTW) makes it feel like it's a randomly failing example

Reply:
As discussed, I'll set the "persmissive" default value to True and remove this "try/except".